data/tr2/ship/cfg/TR2X_strings.json5

```diff
@@ -141,7 +141,7 @@
         {
             "title": "Ice Palace",
             "objects": {
-                "tiger": {"name": "Snow Leopard"},
+                "tiger": {"name": "White Tiger"},
```

Comment:
White Tiger? :D

Reply:
aredfan shrewdly noticed this, so Ice Palace uses different textures compared to the Bengal tiger and snow leopard elsewhere 
![image](https://github.com/user-attachments/assets/7a954c85-91cf-48f4-bc78-6c292eb11100)